graphene_django/filter/utils.py

```diff
@@ -50,7 +51,7 @@ def get_filtering_args_from_filterset(filterset_class, type):
             ):
                 # Get the filter field for filters that are no explicitly declared.
                 if filter_type == "isnull":
-                    field = graphene.Boolean(required=required)
+                    graphene.Boolean(required=required)
```

Comment:
`field = ...` was removed here as it's not referenced later. Was this a typo and should have been `model_field = ...` instead?

Reply:
Oh dang, this shows the value of a good linter! Looking back at the PR that introduced this code https://github.com/graphql-python/graphene-django/pull/1119/files#diff-cdc00f7a891a74f7c16d107ad5346f75d56a657b9316c8b207283c7b507cecb6R52 and trying to understand what this code is doing, I think this line should actually be:

```python
                field_type = graphene.Boolean(required=required)
```

The function is trying to return a graphene type given a filter, and it's saying if the filter is "isnull", then we should be returning a `graphene.Boolean` field for that. That matches up with the rest of the `type`s of the various variables and logically makes sense to me.